.github/workflows/integration-test.yml

```diff
@@ -16,6 +16,9 @@ jobs:
           wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.3.0/latest/linux/x64/tar/builds/opensearch/dist/opensearch-min-2.3.0-linux-x64.tar.gz
           tar -xzf opensearch-*.tar.gz
           rm -f opensearch-*.tar.gz
+
+      - name: Check the validation of OpenSearch Security Plugin
+        run: wget -S --spider https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.3.0/latest/linux/x64/tar/builds/opensearch/plugins/opensearch-security-2.3.0.0.zip || (echo "Please make sure security plugin has been bumped to the same version and added to manifest." && exit 1)
```

Comment:
Is it possible to create variables in the Github actions files? We use the version in a number of places and it would be nice to define it once.

Reply:
Thanks for the reply. I think we can only set up the environment variables in the Github actions files. I can definitely try to set it up in the environment variable to see if it works.